stories/Nested.stories.tsx

```diff
@@ -101,6 +101,110 @@ export const Linking = () => {
   );
 };
 
+export const NestedLinking = () => {
+  const nodeDimensions : any = {
+    typeA: {
+      width: 190,
+      height: 150
+    },
+    typeB:{
+      width: 80,
+      height: 80
+    }
+  }
+  const nodes = [
+    { 
+      id: '1', 
+      text: '1',
+    },
+    { 
+      id: '2',
+      label: 'A',
+      name: 'Process XYZ',
+      description: 'Description of XYZ',
+      //describes padding for nested nodes
```

Comment:
Add space - code editors will format this better if added.

Reply:
Could you please elaborate? A space after the `//`?

Reply:
In some cases, your code editor will do special highlighting on comments but it won't recognize it w/o the space in some cases